README.md

```diff
@@ -173,9 +173,30 @@ If you are using an older version of RN (i.e. `MainApplication.java` does not co
 ...
 ```
 
+* Permissions on Android M and greater
```

Comment:
This is the primary edit.  Only realized post PR that I think my linter removed extra spacing and [tabs](https://www.youtube.com/watch?v=SsoOG6ZeyUI).  Can submit a PR without the spacing if requested.